site/source/cookbook/modern-web-apps/efficient-state-management.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: docs
+category: modern-web-apps
+title: Manage state in your application
+overview: Manage state to decrease complexity in your application
+---
+
+## Objective
+
+This recipe demonstrates a technique for managing state in your Dojo 2 application.
+
+## Procedure
+
+1. Create a widget which uses a property from `this.properties`:
+
+```js
+render() {
+    return v('div', [this.properties.message]);
+}
+```
+
+2. Wrap the widget created in step one with a `Container` by creating the file `src/containers/YourWidgetContainer.ts` and adding the following code:
```

Comment:
Are cookbooks the right place to give a quick explanation of what things are? For example, what does using `app-state` in the `Container` actually mean?

Reply:
Have added an explanation for `app-state`.

Personally I think it's a fine balance between providing minimal code examples and becoming a tutorial. I would say since the cookbook already links off to a detailed tutorial, it's acceptable, but in this case, I didn't see a clarification that the identifier can be string you desire.